plugins/chat/app/services/chat/update_message.rb

```diff
@@ -81,6 +81,8 @@ def can_modify_message(guardian:, message:, **)
     def modify_message(contract:, message:, guardian:, uploads:, **)
       message.message = contract.message
       message.last_editor_id = guardian.user.id
+      message.cooked = ::Chat::Message.cook(contract.message, user_id: guardian.user.id)
+      message.cooked_version = ::Chat::Message::BAKED_VERSION
```

Comment:
Would be good to call `message.cook` instead, it does both these things, cooks the message and sets `cooked_version`:

https://github.com/discourse/discourse/blob/906caa63d777568245a46b88443a8d792678a86f/plugins/chat/app/models/chat/message.rb#L148C1-L155C8

Reply:
yes I hesitated but I think you are right would be better